lib/guardian/schema.graphql

```diff
@@ -0,0 +1,14 @@
+type Guardian {
+  id: ID!
+  firstName: String!
+  middleName: String
+  lastName: String!
+  address: String!
```

Comment:
Good heads up. We probably just want to do change `address` to `addressLine1` and `addressLine2` in this PR then. I don't think it will raise conflicts.

Reply:
Actually, I think we can still merge #24 first, then rebase. This PR doesn't have any database changes, and the GraphQL schema changes depend on the SQL schemas.